inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/multiclass_nms.cpp

```diff
@@ -12,12 +12,12 @@ using namespace LayerTestsDefinitions;
 using namespace InferenceEngine;
 using namespace ngraph;
 
-const std::vector<InputShapeParams> inShapeParams = {InputShapeParams {3, 100, 5}, InputShapeParams {1, 10, 50}, InputShapeParams {2, 50, 50}};
+const std::vector<InputShapeParams> inShapeParams = {InputShapeParams {3, 100, 1}, InputShapeParams {1, 10, 50}, InputShapeParams {2, 50, 50}};
 
 const std::vector<int32_t> nmsTopK = {-1, 20};
 const std::vector<float> iouThreshold = {0.7f};
 const std::vector<float> scoreThreshold = {0.7f};
-const std::vector<int32_t> backgroundClass = {-1, 0};
+const std::vector<int32_t> backgroundClass = {1, 0};
```

Comment:
Hi, @luo-cheng2021 : as backgroundClass=-1 will check different loop branch of the Op implement, and it's also a default value in some framework. so suggest keep this value in test case, change '0' value instead or add new '1' value.

Reply:
Updated.